src/main/java/entities/ApiRequestError.java

```diff
@@ -3,8 +3,8 @@
 /**
```

Comment:
I don't think this is an entity, since it depends on our implementation. It would possibly be better suited as some utility/subpackage in the gateways package

Reply:
I moved the error class to its own package.